components/benefit_sponsors/app/data_migrations/modify_benefit_application.rb

```diff
@@ -18,7 +18,19 @@ def migrate
   end
 
   def update_aasm_state(benefit_applications)
-
+    benefit_application = benefit_applications.where(aasm_state: "enrollment_ineligible").last
+    if benefit_application.present?
+      benefit_application.begin_open_enrollment!
+      benefit_sponsorship = benefit_application.benefit_sponsorship
+      benefit_sponsorship.update_attributes!(aasm_state: "initial_enrollment_open")
```

Comment:
Add a condition to check if benefit application is initial or not....benefit sponsorship state needs to be updated only when application is initial.